app/src/main/java/com/android/sample/ui/authentication/SignUp.kt

```diff
@@ -125,15 +114,7 @@ fun SignUpScreen(navigationActions: NavigationActions) {
                         withStyle(style = SpanStyle(fontStyle = FontStyle.Italic)) { append("now") }
                         append("\nto join our community")
                       },
-                  style =
-                      TextStyle(
-                          fontSize = 32.sp,
-                          lineHeight = 40.sp,
-                          fontFamily = FontFamily(Font(R.font.roboto)),
-                          fontWeight = FontWeight(500),
-                          color = PrimaryPurple,
-                          textAlign = TextAlign.Center,
-                          letterSpacing = 0.32.sp),
+                  style = TypographyBeatLink.displayLarge,
```

Comment:
Consider utilizing MaterialTheme.typography to further centralize theme control and maintain consistency throughout the application.

```suggestion
                  style = MaterialTheme.typography.displayLarge,
```